src/types.ts

```diff
@@ -1,6 +1,16 @@
 import LostLockError from './errors/LostLockError'
 import { Lock } from './Lock'
 
+import type * as ioredis from 'ioredis'
+
+/**
+ * ioredis-like Redis client
+ */
+export type RedisClient = Pick<
+  ioredis.Redis,
+  'script' | 'eval' | 'evalsha' | 'get' | 'set' | 'zrem' | 'options'
```

Comment:
- `script` is not used in code and can be removed
- `options` should be optional, because most likely other clients/mocks will have own interface for options

Reply:
fixed 👍🏽